<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FAboutPlugins</class>
 <widget class="QDialog" name="FAboutPlugins">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugins instalados</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; color:#004040;&quot;&gt;Plugins instalados&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListView" name="lstPlugins">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Desarrollador:</string>
       </property>
       <property name="buddy">
        <cstring>ledDesarrollador</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledDesarrollador"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Descripcion:</string>
       </property>
       <property name="buddy">
        <cstring>txedDescripcion</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="txedDescripcion"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Dependencias:</string>
       </property>
       <property name="buddy">
        <cstring>txedDependencias</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="txedDependencias"/>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnCerrar">
     <property name="text">
      <string>Cerrar</string>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/pics/dialog-close.png</normaloff>:/pics/dialog-close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCerrar</sender>
   <signal>clicked()</signal>
   <receiver>FAboutPlugins</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
